### PR TITLE
Prepare for Go 1.16

### PR DESCRIPTION
**Pull request by rubiojr**
Go 1.16 will make it harder to not use modules by default.

* Adds Go modules support
* Bumped gotk3 and added required fixes

The rest of the changes are related

Note that gotk3 will need to be bumped again for Go 1.16, to include
https://github.com/gotk3/gotk3/pull/687

### CODE REVIEW

File: cmd/xkcd-gtk/style.go [hunk 1]
```diff
@@ -157,11 +158,11 @@ func (win *Window) StyleUpdated() {
 		}
 	}
 
-	menuPopoverChild, err := win.menu.GetPopover().GetChild()
+	c, err := win.menu.GetPopover().GetChild()
 	if err != nil {
 		log.Print(err)
 	} else {
-		menuBox := (&gtk.Stack{Container: gtk.Container{Widget: *menuPopoverChild}}).GetVisibleChild()
+		menuBox := c.ToWidget()
```
Comment (rubiojr) [inline cmd/xkcd-gtk/style.go:165]: I'm not entirely sure about this change. Seems to work as expected, but didn't understand the motivation behind the original code.

Comment (rkoesters) [reply cmd/xkcd-gtk/style.go:165]: This code isn't great and I ended up removing it in head due to version difficulties maintaining it. I might reintroduce it later before the next release, but for now it is gone.

File: cmd/xkcd-gtk/properties-dialog.go [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
 import (
-	"github.com/gotk3/gotk3/gdk"
-	"github.com/gotk3/gotk3/gtk"
 	"log"
 	"strconv"
 	"strings"
 	"time"
+
+	"github.com/gotk3/gotk3/gdk"
```
Comment (rkoesters) [inline cmd/xkcd-gtk/properties-dialog.go:9]: Is separating url imports out a manual choice, or do newer version of go-fix do that?

**Comment by rkoesters** [comment]
Without reviewing your PR, I already worked on go modules support in parallel, although I targeted Go 1.13 since that is the version used by ubuntu focal.

**Pull request closed without merging**